scripts/serverless.js

```diff
@@ -901,5 +909,6 @@ const processSpanPromise = (async () => {
     });
   } finally {
     clearTimeout(keepAliveTimer);
+    log.notice(); // Ensure breathing line at the end of output
```

Comment:
Should we have that in? I generally like the idea and I think we could use it in more places, but I remember that e.g. @mnapoli mentioned that we should not do it as people might have terminals configured to automatically print/render empty line after commands.

Reply:
Good point, I just assumed it's good to have that (design docs are not explicit on intention, as they do not show prompt after finalized output). Anyway let's first clarify with @mnapoli

Reply:
As we discussed on Slack, currently there's an agreement to add this in (but it's possible we'll revert that)

Reply:
Actually reverting that (following further discission on slack)